admin/connect.php

```diff
@@ -0,0 +1,86 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: VALEBLES
+ * Date: 05/02/2019
+ * Time: 16:14
+ */
+
+// connection to database
+require ('../include/config.php');
+
+// we define if the form has been validated
+if(isset($_POST['submit'])) {
+
+    // initialization of error message
+    $errMsg = "";
+
+    // we define the variable with the data filled
+    if (isset($_POST['username']) AND isset($_POST['password'])) {
+        $username = $_POST['username'];
+        $password = $_POST['password'];
+    }
+
+    // we prepare the data which correspond to data filled
+    $query = $bdd->prepare('SELECT * FROM user WHERE username = :username');
+    $query->execute(array(
+        ':username' => $username
+    ));
+    $data = $query->fetch();
+
+    // compare the results
+    if ($data !== 0) {
+        if ($password == $data['password']) {
```

Comment:
Ne fais pas une nouvelle condition isi. Tu devrais directement regarder dans la requête en une fois avec 

where username = :username and password = :key: 

Ca t'évitera cette étape ;)

Reply:
Done